In [ ]:
#1
# Define the points
x1, y1 = 1, 2  # Point A
x2, y2 = 3, 4  # Point B

# Calculate the slope (m)
m = (y2 - y1) / (x2 - x1)

# Find the y-intercept (b) using the slope-intercept formula: y = mx + b
b = y1 - m * x1

# Print the equation of the line
print(f"The equation of the line is: y = {m}x + {b}")


The equation of the line is: y = 1.0x + 1.0


To find the equation of the line passing through points $ A(1, 2) $ and $ B(3, 4) $:

### Step 1: Calculate the slope $\ m \$
$
m = \frac{y_2 - y_1}{x_2 - x_1} = \frac{4 - 2}{3 - 1} = \frac{2}{2} = 1
$

### Step 2: Use the point-slope formula
Using point $ A(1, 2) $:
$
y - y_1 = m(x - x_1) \implies y - 2 = 1(x - 1)
$

### Step 3: Simplify to slope-intercept form
$
y - 2 = x - 1 \implies y = x + 1
$

### Verification:
- For $ A(1, 2) $: $ 2 = 1 + 1 \quad \checkmark $
- For $ B(3, 4) $: $ 4 = 3 + 1 \quad \checkmark $

### Final Answer:
$
\boxed{y = x + 1}
$

In [ ]:
#2
# Define the given point
x1, y1 = 1, 2  # Point A

# Define the slope of the parallel line (same as the given line)
m = 2  # Slope of the line y = 2x + 3

# Use the point-slope formula: y - y1 = m(x - x1)
# Rearrange to get the slope-intercept form: y = mx + b
b = y1 - m * x1

# Print the equation of the line
print(f"The equation of the line is: y = {m}x + {b}")


The equation of the line is: y = 2x + 0


To find the equation of the line passing through point $ A(1, 2) $ and parallel to $ y = 2x + 3 $:

### Step 1: Identify the slope
Parallel lines share the same slope.  
Given line: $ y = 2x + 3 $  
Slope $ m \ = $ 2 .

### Step 2: Use the point-slope formula
Substitute $ m = 2 $ and point $ (1, 2) $:  
$
y - y_1 = m(x - x_1) \implies y - 2 = 2(x - 1)
$

### Step 3: Simplify to slope-intercept form
$
y - 2 = 2x - 2 \implies y = 2x
$

### Verification:
For point $ A(1, 2) $:  
$
2 = 2(1) \quad \checkmark
$

### Final Answer:
$
\boxed{y = 2x}
$

In [ ]:
#3
# Define the given point
x1, y1 = 1, 2  # Point A

# Define the slope of the perpendicular line
# Slope of the given line y = 2x + 3 is 2
# The slope of a perpendicular line is the negative reciprocal
m_perpendicular = -1 / 2

# Use the point-slope formula: y - y1 = m(x - x1)
# Rearrange to get the slope-intercept form: y = mx + b
b = y1 - m_perpendicular * x1

# Print the equation of the line
print(f"The equation of the line is: y = {m_perpendicular}x + {b}")


The equation of the line is: y = -0.5x + 2.5


To find the equation of the line passing through point $ A(1, 2) $ and perpendicular to $ y = 2x + 3 $:

### Step 1: Determine the slope of the perpendicular line  
Perpendicular lines have slopes that are negative reciprocals.  
Given line: $ y = 2x + 3 $  
Slope of given line $$ m_1 \$ = $ 2 $.  
Slope of perpendicular line $ m_2 \$ = $ -\frac{1}{2} $.

### Step 2: Use the point-slope formula  
Substitute $ m = -\frac{1}{2} $ and point $ (1, 2) $:  
$
y - y_1 = m(x - x_1) \implies y - 2 = -\frac{1}{2}(x - 1)
$

### Step 3: Simplify to slope-intercept form  
$
y - 2 = -\frac{1}{2}x + \frac{1}{2} \implies y = -\frac{1}{2}x + \frac{5}{2}
$

### Verification:  
For point \( A(1, 2) \):  
$
2 = -\frac{1}{2}(1) + \frac{5}{2} \implies 2 = 2 \quad \checkmark
$

### Final Answer:  
$
\boxed{y = -\frac{1}{2}x + \frac{5}{2}}
$

In [ ]:
#4
import math

# Define the slopes of the lines
m1 = 2  # Slope of line 1 (y = 2x + 3)
m2 = 3  # Slope of line 2 (y = 3x + 2)

# Solve for the intersection point
# Line equations: y = 2x + 3 and y = 3x + 2
# At intersection, 2x + 3 = 3x + 2
# Rearrange to find x
x_intersection = (3 - 2) / (3 - 2)
y_intersection = 2 * x_intersection + 3

# The intersection point
intersection_point = (x_intersection, y_intersection)

# Calculate the angle between the two lines
tan_theta = abs((m1 - m2) / (1 + m1 * m2))
theta_radians = math.atan(tan_theta)  # Angle in radians
theta_degrees = math.degrees(theta_radians)  # Convert to degrees

# Print the results
print(f"The intersection point of the lines is: {intersection_point}")
print(f"The angle between the lines is approximately: {theta_degrees:.2f} degrees")



The intersection point of the lines is: (1.0, 5.0)
The angle between the lines is approximately: 8.13 degrees


To find the intersection point and the angle between the lines \( y = 2x + 3 \) and \( y = 3x + 2 \):

---

### **1. Intersection Point**
**Step 1:** Set the equations equal to solve for \( x \):
$
2x + 3 = 3x + 2 \implies x = 1
$

**Step 2:** Substitute \( x = 1 \) into \( y = 2x + 3 \):
$
y = 2(1) + 3 = 5
$

**Intersection Point:** \(\boxed{(1, 5)}\)

---

### **2. Angle Between the Lines**
**Step 1:** Identify the slopes:
- Slope of \( y = 2x + 3 \): \( m_1 = 2 \)
- Slope of \( y = 3x + 2 \): \( m_2 = 3 \)

**Step 2:** Use the angle formula:
$
\tan\theta = \left| \frac{m_2 - m_1}{1 + m_1 m_2} \right| = \left| \frac{3 - 2}{1 + (2)(3)} \right| = \frac{1}{7}
$

**Step 3:** Solve for $ \theta $:
$
\theta = \arctan\left\rac{1}{7}\right) \approx 8.13^\circ
$

**Angle Between the Lines:** $\boxed{8.13^\circ}$

---

**Verification:**
- **Intersection Point:** Both lines yield \( y = 5 \) when \( x = 1 \).
- **Angle Calculation:** The small angle aligns with the closeness of the slopes (2 and 3).

In [ ]:
#5
# Define the given point
x1, y1 = 1, 2  # Point A

# Define the slope of the line parallel to the vector v = [2, 3]
# The slope is the ratio of the change in y to the change in x
m = 3 / 2  # Slope of the line

# Use the point-slope formula: y - y1 = m(x - x1)
# Rearrange to get the slope-intercept form: y = mx + b
b = y1 - m * x1

# Print the equation of the line
print(f"The equation of the line is: y = {m}x + {b}")


The equation of the line is: y = 1.5x + 0.5


To find the equation of the line passing through point $ A(1, 2) $ and parallel to the vector $ \mathbf{v} = [2, 3] $:

---

### **Step 1: Determine the slope from the vector**  
The direction vector $ \mathbf{v} = [2, 3] $ corresponds to a slope:  
$
m = \frac{\Delta y}{\Delta x} = \frac{3}{2}
$

---

### **Step 2: Use the point-slope formula**  
Substitute $ m = \frac{3}{2} $ and point $ (1, 2) $:  
$
y - y_1 = m(x - x_1) \implies y - 2 = \frac{3}{2}(x - 1)
$

---

### **Step 3: Simplify to slope-intercept form**  
$
y - 2 = \frac{3}{2}x - \frac{3}{2} \implies y = \frac{3}{2}x + \frac{1}{2}
$

---

### **Final Answer**  
$
\boxed{y = \frac{3}{2}x + \frac{1}{2}}
$

---

**Verification:**  
- For \( x = 1 \):  
  $
  y = \frac{3}{2}(1) + \frac{1}{2} = 2 \quad \checkmark
  $
- **Parametric form** (optional):  
  $
  x = 1 + 2t, \quad y = 2 + 3t
  $  
  Eliminating \( t \) confirms the same equation.

In [ ]:
#6
# Define the slope of the given line
m_given = 2  # Slope of the given line

# Parallel line: Same slope, different y-intercept
m_parallel = m_given
b_parallel = 5  # Example y-intercept
parallel_line = f"y = {m_parallel}x + {b_parallel}"

# Perpendicular line: Slope is the negative reciprocal
m_perpendicular = -1 / m_given
b_perpendicular = 1  # Example y-intercept
perpendicular_line = f"y = {m_perpendicular}x + {b_perpendicular}"

# Print the results
print(f"An example of a parallel line: {parallel_line}")
print(f"An example of a perpendicular line: {perpendicular_line}")


An example of a parallel line: y = 2x + 5
An example of a perpendicular line: y = -0.5x + 1


To find examples of lines **parallel** and **perpendicular** to $ y = 2x + 3 $:

---

### **1. Parallel Line**  
Parallel lines share the **same slope** as the original line.  
- Given slope: $ m = 2 $.  
- Example equation (using a different y-intercept):  
  $
  y = 2x + 5
  $  
**Verification**:  
- Slopes are identical $ 2 = 2 $.  
- Lines never intersect (different intercepts).  

---

### **2. Perpendicular Line**  
Perpendicular lines have slopes that are **negative reciprocals**.  
- Original slope: $ m = 2 $.  
- Perpendicular slope: $ m_{\perp} = -\frac{1}{2} $.  
- Example equation:  
  $
  y = -\frac{1}{2}x + 4
  $  
**Verification**:  
- Product of slopes: $ 2 \times \left(-\frac{1}{2}\right) = -1 $, confirming perpendicularity.  

---

### **Final Answer**  
- **Parallel Line**: $\boxed{y = 2x + 5}$
- **Perpendicular Line**: $\boxed{y = -\frac{1}{2}x + 4}$  

In [ ]:
#7
import math

# 1. Equation of the perpendicular line
# Given line slope
m_given = 2

# Slope of the perpendicular line
m_perpendicular = -1 / m_given

# Point A
x1, y1 = 1, 2

# Point-slope form: y - y1 = m(x - x1)
b_perpendicular = y1 - m_perpendicular * x1  # Solve for b
perpendicular_line = f"y = {m_perpendicular}x + {b_perpendicular}"

print(f"Equation of the perpendicular line: {perpendicular_line}")

# 2. Intersection of the two lines
# Equations:
# Line 1: y = 2x + 3
# Line 2: y = -1/2x + 5/2

# Solving 2x + 3 = -1/2x + 5/2
# Rearrange: (5/2)x = -1/2
x_intersection = -1 / 5

# Substitute x into the first line: y = 2x + 3
y_intersection = 2 * x_intersection + 3

intersection_point = (x_intersection, y_intersection)
print(f"Intersection point: {intersection_point}")

# 3. Distance from point A to the intersection point
x2, y2 = intersection_point

# Distance formula: sqrt((x2 - x1)^2 + (y2 - y1)^2)
distance = math.sqrt((x2 - x1)**2 + (y2 - y1)**2)
print(f"Distance from point A to the intersection point: {distance:.2f}")


Equation of the perpendicular line: y = -0.5x + 2.5
Intersection point: (-0.2, 2.6)
Distance from point A to the intersection point: 1.34


To find the distance from point \( A(1, 2) \) to the line $ y = 2x + 3 $:

---

### **Step 1: Rewrite the line in standard form**
$
y = 2x + 3 \implies -2x + y - 3 = 0
$
Here, \( A = -2 \), $ B = 1 $, and $ C = -3 $.

---

### **Step 2: Apply the distance formula**
The distance $ d $ from a point $ (x_0, y_0) $ to the line $ Ax + By + C = 0 $ is:
$
d = \frac{|A x_0 + B y_0 + C|}{\sqrt{A^2 + B^2}}
$
Substitute $ A = -2 $, $ B = 1 $, $ C = -3 $, $ x_0 = 1 $, $ y_0 = 2 $:
$
d = \frac{|-2(1) + 1(2) - 3|}{\sqrt{(-2)^2 + 1^2}} = \frac{|-2 + 2 - 3|}{\sqrt{5}} = \frac{3}{\sqrt{5}}
$

---

### **Step 3: Rationalize the denominator**
$
d = \frac{3}{\sqrt{5}} \times \frac{\sqrt{5}}{\sqrt{5}} = \frac{3\sqrt{5}}{5}
$

---

### **Verification Using Perpendicular Intersection**
1. **Equation of the perpendicular line** through $ A(1, 2) $:  
   Slope of perpendicular line = $ -\frac{1}{2} $.  
   Using point-slope form:  
   $
   y - 2 = -\frac{1}{2}(x - 1) \implies y = -\frac{1}{2}x + \frac{5}{2}
   $

2. **Find intersection** with $ y = 2x + 3 $:  
   Solve $ 2x + 3 = -\frac{1}{2}x + \frac{5}{2} $:  
   $
   \frac{5}{2}x = -\frac{1}{2} \implies x = -\frac{1}{5}, \quad y = \frac{13}{5}
   $

3. **Distance between $ A(1, 2) $ and $ \left(-\frac{1}{5}, \frac{13}{5}\right) $**:
   $
   \sqrt{\left(1 + \frac{1}{5}\right)^2 + \left(2 - \frac{13}{5}\right)^2} = \sqrt{\left(\frac{6}{5}\right)^2 + \left(-\frac{3}{5}\right)^2} = \frac{3\sqrt{5}}{5}
   $

---

### **Final Answer**
The distance from $ A(1, 2) $ to the line $ y = 2x + 3
$
$
\boxed{\frac{3\sqrt{5}}{5}}
$

In [ ]:
#8
# Define the points A and B
x1, y1 = 2, 0  # Point A
x2, y2 = 0, 3  # Point B

# Calculate the slope (m)
m = (y2 - y1) / (x2 - x1)

# Find the y-intercept (b) using the slope-intercept formula y = mx + b
b = y1 - m * x1

# Print the equation of the line
print(f"The equation of the line is: y = {m}x + {b}")


The equation of the line is: y = -1.5x + 3.0


To find the equation of the line intersecting the coordinate axes at $ A(2, 0) $ and $ B(0, 3) $:

---

### **Method 1: Intercept Form**  
The intercept form of a line is:  
$
\frac{x}{a} + \frac{y}{b} = 1
$  
where $ a = 2 $ (x-intercept) and $ b = 3 $ (y-intercept).  
Substitute \( a \) and \( b \):  
$
\frac{x}{2} + \frac{y}{3} = 1
$

---

### **Method 2: Slope-Intercept Form**  
1. **Calculate the slope (\( m \))** using the two points:  
   $
   m = \frac{y_2 - y_1}{x_2 - x_1} = \frac{3 - 0}{0 - 2} = -\frac{3}{2}
   $

2. **Use the point-slope formula** with $ A(2, 0) $:  
   $
   y - y_1 = m(x - x_1) \implies y - 0 = -\frac{3}{2}(x - 2)
   $

3. **Simplify**:  
   $
   y = -\frac{3}{2}x + 3
   $

---

### **Verification**  
- **At $ x = 2 $:**  
  $
  y = -\frac{3}{2}(2) + 3 = 0 \quad \checkmark \quad (A(2, 0))
  $
- **At \( x = 0 \):**  
  $
  y = -\frac{3}{2}(0) + 3 = 3 \quad \checkmark \quad (B(0, 3))
  $

---

### **Final Answer**  
The equation of the line is:  
$
\boxed{y = -\frac{3}{2}x + 3}
$

In [ ]:
#9
import math

# Slope of the line
m = 1  # Slope of y = x + 3

# Calculate the angle in radians using arctan
theta_radians = math.atan(m)

# Convert the angle to degrees
theta_degrees = math.degrees(theta_radians)

# Print the angle
print(f"The angle between the line and the x-axis is approximately {theta_degrees:.2f} degrees.")


The angle between the line and the x-axis is approximately 45.00 degrees.


## Calculating the Angle Between the Line and the \( x \)-Axis

The equation of the given line is:

$
y = x + 3
$

### Step 1: Find the Slope
The equation is in the slope-intercept form $ y = mx + c $, where $ m $ is the slope. From the given equation, we see that:

$
m = 1
$

### Step 2: Find the Angle with the \( x \)-Axis
The angle $ \theta $ between the line and the positive direction of the $ x $-axis is given by:

$
\theta = \tan^{-1}(m)
$

Substituting $ m = 1 $:

$
\theta = \tan^{-1}(1)
$

$
\theta = 45^\circ
$

### Final Answer:
The angle between the given line and the $ x $-axis is **45 degrees**.


In [ ]:
#10
# Given line equation: x + y + 1 = 0
# Standard form: Ax + By + C = 0
# A = 1, B = 1 (coefficients directly give the perpendicular vector to the line)

# Define the coefficients A and B
A = 1
B = 1

# Perpendicular vector to the line
perpendicular_vector = (A, B)

# Print the result
perpendicular_vector

(1, 1)

To find a vector perpendicular to the line \$x + y + 1 = 0 $:

1. **General Form of a Line**:  
   The equation $ Ax + By + C = 0 $ has a normal vector $ \begin{pmatrix}A \\ B\end{pmatrix} $.  
   For $ x + y + 1 = 0 $, the coefficients are $ A = 1 $ and $ B = 1 $, so the normal vector is:  
   $
   \begin{pmatrix}1 \\ 1\end{pmatrix}.
   $

2. **Verification via Slope**:  
   Rewrite the line as $ y = -x - 1 $.  
   - Slope of the line: $ -1 $.  
   - Slope of the normal vector (negative reciprocal): $ 1 $, which matches the vector $ \begin{pmatrix}1 \\ 1\end{pmatrix} $.

3. **Direction Vector Check**:  
   Two points on the line: $ (0, -1) $ and $ (-1, 0) $.  
   Direction vector: $ \begin{pmatrix}-1 \\ 1\end{pmatrix} $.  
   Dot product with $ \begin{pmatrix}1 \\ 1\end{pmatrix} $:  
   $
   1 \cdot (-1) + 1 \cdot 1 = -1 + 1 = 0 \quad (\text{perpendicular}).
   $

**Final Answer**:  
The vector perpendicular to the line is $\boxed{\begin{pmatrix}1 \\ 1\end{pmatrix}}$.